prov/rxm/src/rxm_ep.c

```diff
@@ -962,7 +962,7 @@ rxm_ep_alloc_rndv_tx_res(struct rxm_ep *rxm_ep, struct rxm_conn *rxm_conn, void
 	}
 
 	rxm_ep_format_tx_buf_pkt(rxm_conn, data_len, op, data, tag, flags, &(tx_buf)->pkt);
-	tx_buf->pkt.ctrl_hdr.msg_id = rxm_tx_buf_2_msg_id(tx_buf);
+	tx_buf->pkt.ctrl_hdr.msg_id = rxm_tx_buf_2_msg_id(rxm_ep, RXM_BUF_POOL_TX_RNDV, tx_buf);
```

Comment:
Also, this commit should be squashed with the one that changes the msg_id first so that we don't have a broken build between commits.

Reply:
This commit meld into https://github.com/ofiwg/libfabric/pull/4554/commits/c14eb07bb734c4275d1492267e6423b0e02d64fc commit